buildsupport/testing/pom.xml

```diff
@@ -136,6 +136,12 @@
         <artifactId>junit</artifactId>
         <version>4.11</version>
       </dependency>
+
+      <dependency>
+        <groupId>javax.transaction</groupId>
+        <artifactId>javax.transaction-api</artifactId>
+        <version>1.2</version>
+      </dependency>
```

Comment:
This isn't really a testing dependency, though it may only be used as a test, I'd move this to another buildsupport group.